CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.10)
 
 project(bzapi CXX)
 
+# for now...
```

Comment:
I assume this means we will move to a separate CMakeLists.txt in the future as we do with SwarmDB

Reply:
@amastracci Then the source code will also need the licence cleaned up. @paularchard please create a task for this.

Reply:
yes... eventually.